prisma/migrations/20230405054652_postgres_init/migration.sql

```diff
@@ -0,0 +1,93 @@
+-- CreateEnum
```

Comment:
huh was this missing from server?  be careful about migrations, i think this is out of scope of the check you were producing

Can we make a PR without the migration please?  I'm just concerned that since we manually migrated the server already that this might cause problems when we deploy